R/JointModel.R

```diff
@@ -0,0 +1,48 @@
+cmdstan_mod <- R6::R6Class("CmdStanModel")
+setOldClass("CmdStanModel")
+
+#' @exportClass JointModel
+JointModel <- setClass("JointModel",
+    slots = c(cmdstan_mod = "CmdStanModel")
+)
+
+
+
+#' @importFrom cmdstanr write_stan_file
+#' @importFrom cmdstanr cmdstan_model
+#' @rdname StanModule-class
+#' @param util Logical. Populates the .stan file with general functions.
+#' @export
+setMethod(
+    f = "initialize",
+    signature = "JointModel",
+    definition = function(.Object,
+                          ...,
+                          long,
+                          os,
+                          util = TRUE,
+                          name = "stan_model.stan") {
+        mod <- merge(x = long, y = os)
+
+        utils <- StanModule(functions = read_file(system.file("Utils", "utils.stan", package = "jmpost")))
```

Comment:
Maybe we should consider making the `read_stan` function more generic. Right now it only can search files into the LogLogistic folder of the package.

Reply:
yes please , that should be more generic

Reply:
maybe do a separate issue and PR for that though